src/sentry/static/sentry/app/components/charts/baseChart.tsx

```diff
@@ -43,6 +44,21 @@ const getDimensionValue = (dimension?: ReactEChartOpts['height']) => {
   return dimension;
 };
 
+// TODO(ts): What is the series type? EChartOption.Series's data cannot have
+// `onClick` since it's typically an array.
```

Comment:
I'm pretty sure these click handlers are something we've spliced on top of echarts.

Reply:
This just moved up, but yeah I think you're right. Need to investigate